library.js

```diff
@@ -225,7 +225,7 @@ plugin.getUsers = function (callback) {
 
 plugin.updateTitle = function (data, callback) {
 	translator.translate('[[2factor:title]]', function (title) {
-		if (data.templateData.url.match(/user\/.+\/2factor/)) {
+		if (Array.isArray(data.templateData.url.match(/user\/.+\/2factor/))) {
```

Comment:
Use of `Array.isArray` is potentially confusing, can you please remove that and instead use `!!(data.templateData.url.match(/user\/.+\/2factor/).length)`?

Reply:
I think even the `!!` is not needed, and can be excluded.